.github/workflows/sync_branches.yaml

```
@@ -49,7 +49,7 @@ jobs:
             // Create a new Octokit instance under Apify Service Account
             // Apify Service Account does not trigger PR toolkit,
             // does not need approval for PRs and does not need to wait for status checks
-            const octokit = new Octokit({
+            const octokitExt = new Octokit({
```

Let's call it something more explicit, like `serviceAccountOctokit`:
```suggestion
            const serviceAccountOctokit = new Octokit({
```